### ISSUE TITLE
User-controlled accessibility options such as background and text colour

### ISSUE THREAD

**Issue by christianp** [numbas/Numbas#797]
In the default theme there should be controls for common accessibility adjustments:

* Background colour
* Text colour
* Text font

I don't think we need to adjust text size, because you can use the browser's zoom. I'll check if that's common practice.

**Comment by christianp** [comment]
I have a theme adding these options at https://numbas.mathcentre.ac.uk/themes/343/edit. It might be that the changes in this theme can just be part of the default theme.
